bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -190,67 +196,63 @@ public void removeEntryLog(long logToRemove) {
             }
             if (majorCompactionInterval <= gcWaitTime) {
-                throw new IOException("Too short major compaction interval : "
-                                    + majorCompactionInterval);
+                throw new IOException("Too short major compaction interval : " + majorCompactionInterval);
             }
             enableMajorCompaction = true;
         }
 
         if (enableMinorCompaction && enableMajorCompaction) {
             if (minorCompactionInterval >= majorCompactionInterval
-                || minorCompactionThreshold >= majorCompactionThreshold) {
-                throw new IOException("Invalid minor/major compaction settings : minor ("
-                                    + minorCompactionThreshold + ", " + minorCompactionInterval
-                                    + "), major (" + majorCompactionThreshold + ", "
-                                    + majorCompactionInterval + ")");
+                    || minorCompactionThreshold >= majorCompactionThreshold) {
+                throw new IOException("Invalid minor/major compaction settings : minor (" + minorCompactionThreshold
+                        + ", " + minorCompactionInterval + "), major (" + majorCompactionThreshold + ", "
+                        + majorCompactionInterval + ")");
             }
         }
 
-        LOG.info("Minor Compaction : enabled=" + enableMinorCompaction + ", threshold="
-               + minorCompactionThreshold + ", interval=" + minorCompactionInterval);
-        LOG.info("Major Compaction : enabled=" + enableMajorCompaction + ", threshold="
-               + majorCompactionThreshold + ", interval=" + majorCompactionInterval);
+        LOG.info("Minor Compaction : enabled=" + enableMinorCompaction + ", threshold=" + minorCompactionThreshold
+                + ", interval=" + minorCompactionInterval);
+        LOG.info("Major Compaction : enabled=" + enableMajorCompaction + ", threshold=" + majorCompactionThreshold
+                + ", interval=" + majorCompactionInterval);
 
         lastMinorCompactionTime = lastMajorCompactionTime = System.currentTimeMillis();
     }
 
     public void enableForceGC() {
         if (forceGarbageCollection.compareAndSet(false, true)) {
             LOG.info("Forced garbage collection triggered by thread: {}", Thread.currentThread().getName());
-            triggerGC(true, suspendMajorCompaction.get(),
-                      suspendMinorCompaction.get());
+            triggerGC(true, suspendMajorCompaction.get(), suspendMinorCompaction.get());
         }
     }
 
     public void disableForceGC() {
         if (forceGarbageCollection.compareAndSet(true, false)) {
-            LOG.info("{} disabled force garbage collection since bookie has enough space now.", Thread
-                    .currentThread().getName());
+            LOG.info("{} disabled force garbage collection since bookie has enough space now.",
```

Comment:
it would be better not to reformat code, as this changes makes the diff harder to review.

Can you revert the formatting ?
as soon as checkstyle is happy we are okay

Reply:
yes, formatting was not intentional change but IDE performed automatically. i will try to revert these line changes.